library/lib_window.ks

```diff
@@ -0,0 +1,26 @@
+
+
+@lazyglobal off.
+
+function create_window{
```

Comment:
Stop saying "[done and commited]" please. Can look at the code myself. And this spams my notification list.